pyomo/gdp/util.py

```diff
@@ -124,7 +162,7 @@ def _visit_vertex(self, u, leaf_to_root):
 
     def _topological_sort(self):
         # this is reverse of the list we should return (but happens to be what
-        # we want for hull and bigm)
+        # we want for bigm)
```

Comment:
Should the really be called `_topological_sort`?  It is confusing that `topological_sort` returns the reverse of this generator.  What about renaming it to `_reverse_topological_iterator`

Reply:
Hehehe, good point. I'm all for not naming a method as the opposite of what it does... :P